src/test/java/org/broadinstitute/hellbender/tools/copynumber/GermlineCNVCallerIntegrationTest.java

```diff
@@ -0,0 +1,83 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.tools.copynumber.arguments.CopyNumberStandardArgument;
+import org.broadinstitute.hellbender.utils.IntervalMergingRule;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.stream.IntStream;
+
+/**
+ * Integration tests for {@link GermlineCNVCaller}.
+ */
+public final class GermlineCNVCallerIntegrationTest extends CommandLineProgramTest {
+    private static final String gCNVSimDataDir = copyNumberTestDir + "gcnv-sim-data/";
+    private static final File[] testCountFiles = IntStream.range(0, 20)
+            .mapToObj(n -> new File(gCNVSimDataDir + String.format("SAMPLE_%03d_counts.tsv", n)))
```

Comment:
Same thing about `new File...` here and below.

Reply:
Done.